packages/backend/src/api/apiV1.ts

```diff
@@ -26,7 +29,7 @@ apiV1Router.get('/health', async (req, res, next) => {
 
 apiV1Router.post('/register', unauthorisedInDemo, async (req, res, next) => {
     try {
-        const lightdashUser = await UserModel.register({
+        const lightdashUser = await userService.registerInitialUser({
```

Comment:
@ZeRego we only use the `/register` endpoint for the initial user correct?

Reply:
Correct 👍